v2/pkg/protocols/common/utils/vardump/dump.go

```diff
@@ -0,0 +1,34 @@
+package vardump
+
+import (
+	"strings"
+
+	"github.com/projectdiscovery/gologger"
+	"github.com/projectdiscovery/nuclei/v2/pkg/types"
+)
+
+// Variables writes the truncated dump of variables on the stderr
+// in a formatted key-value manner using gologger.
+//
+// The values are truncated to return 50 characters from start and end.
+func Variables(data map[string]interface{}) {
```

Comment:
Function names should be verbs. Something like `DumpVariables` or `LogVariables` or something.